src/bitcoind/mod.rs

```diff
@@ -187,6 +194,43 @@ fn wallet_transaction(bitcoind: &BitcoinD, txid: Txid) -> Option<WalletTransacti
         .ok()
 }
 
+fn cpfp(
+    revaultd: Arc<RwLock<RevaultD>>,
+    bitcoind: Arc<RwLock<BitcoinD>>,
+    txids: Vec<Txid>,
+    feerate: f64,
+) -> Result<(), BitcoindError> {
+    let db_path = revaultd.read().unwrap().db_file();
+    assert!(revaultd.read().unwrap().is_manager());
+
+    let mut cpfp_txs = Vec::with_capacity(txids.len());
+
+    for txid in txids.iter() {
+        let spend_tx = db_spend_transaction(&db_path, &txid).expect("Database must be available");
+
+        if let Some(unwrap_spend_tx) = spend_tx {
+            // If the transaction is of type SpendTransaction
+            cpfp_txs.push(ToBeCpfped::Spend(unwrap_spend_tx.psbt));
+        } else {
+            // The transaction type is asserted to be UnvaultTransaction
+            let unvault_tx = db_vault_by_unvault_txid(&db_path, &txid)
+                .expect("Database must be available")
+                .unwrap()
+                .1
+                .psbt
+                .assert_unvault();
```

Comment:
Hmmmm that sounds like this line may panic if you are passed a txid that does not point to anything :)